src/lib/localStorage.ts

```diff
@@ -51,10 +51,12 @@ export enum StorageKey {
 }
 
 export interface Settings {
+	apiType: 'ollama' | 'openai';
 	ollamaServer: string | null;
 	ollamaModel: string | null;
 	ollamaModels: ModelResponse[];
 	ollamaServerStatus: 'connected' | 'disconnected';
```

Comment:
I don't think we want to store an `apiType` in settings, in fact, I think we should get rid of `ollamaModel` as well because after #193 you can switch models at any time.

The chosen model is what's going to determine if we hit the Ollama or OpenAI APIs.

By default the chosen model would be whichever is the last one you used for the **current session**.

Then, when you start a **new session** the default model should be the one at the top of the list in `Recently used`:
<img width="706" alt="image" src="https://github.com/user-attachments/assets/06be5e3f-1e15-4883-a5da-cb5124effdf2">

Or no model selected if there are no sessions.